contracts/pm/mixins/MixinTicketBrokerCore.sol

```diff
@@ -211,7 +198,7 @@ contract MixinTicketBrokerCore is MReserve, MTicketProcessor, MTicketBrokerCore
     /**
      * @dev Withdraws all ETH from the caller's deposit and reserve
      */
-    function withdraw() public reserveNotFrozen(msg.sender) {
+    function withdraw() public {
         Sender storage sender = senders[msg.sender];
 
         uint256 deposit = sender.deposit;
```

Comment:
Even though not part of this PR, we don't need this additional MSTORE operation, we can just SLOAD the deposit directly using `sender.deposit` on L208